src/pages/version/taskDuration/TaskDurationRow.tsx

```diff
@@ -73,21 +74,23 @@ TaskDurationRow.displayName = "Row";
 const calculateBarWidth = (value: number, max: number) =>
```

Comment:
oh no... this may be another ticket. 😔 I see that the display tasks won't expand anymore in this table...

<img width="1130" alt="Screen Shot 2022-12-02 at 12 25 44 PM" src="https://user-images.githubusercontent.com/47064971/205350167-30893941-cf18-4342-8590-844e628edba1.png">

Reply:
EVG-18449